src/main/java/com/mraof/minestuck/event/ServerEventHandler.java

```diff
@@ -154,6 +155,13 @@ public static void onEntityAttack(LivingHurtEvent event)
 					double modifier = PlayerSavedData.getData(player).getEcheladder().getUnderlingDamageModifier();
 					event.setAmount((float) (event.getAmount() * modifier));
 				}
+				
+				Title title = PlayerSavedData.getData(player).getTitle();
+				boolean isVoid = title != null && title.getHeroAspect() == EnumAspect.VOID;
+				if (player.getHeldItemMainhand().getItem() == MSItems.DESOLATOR_MACE && isVoid)
```

Comment:
I'm not sure if this accurately catches only damage dealt by the weapon, as all we've confirmed about the damage source is that the player caused it. There could be other ways for players to be a source of damage other than direct weapon attacks.

Also, it'd be better design to go through some property on the weapon item instead of hardcoding this here.

Reply:
What if it checks if the active item is the same as well?

I would make it a weapon property, although I did not want to have to override the damage value and I did not want to make it give damage in addition to what the player is already inflicting, I wanted it to override the damage correctly. If there is a succinct way of changing it, it is absolutely worth doing so.

Reply:
Issue mitigated by the use of a OnHitEffect damage bonus instead of controlling the action damage action